server/@types/models/Referral.ts

```diff
@@ -11,6 +11,7 @@ type Referral = {
   offeringId: CourseOffering['id']
   prisonNumber: Person['prisonNumber']
   referrerId: Express.User['userId']
```

Comment:
Do we need `referrerId` anymore? Can we remove it in this PR?

Reply:
I'll do that as a separate PR as it's an additional 9 file change.